packages/component-base/src/virtualizer-iron-list-adapter.js

```diff
@@ -143,6 +143,35 @@ export class IronListAdapter {
     });
   }
 
+  _updateMetrics(itemSet) {
```

Comment:
Please remove the function from `iron-list-core.js` if it's no longer needed (could add a mention to the [file](https://github.com/vaadin/web-components/blob/master/packages/component-base/src/iron-list-core.js#L18-L28) that dead code removals are ok).